streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/ThreadMetrics.java

```diff
@@ -290,6 +295,30 @@ public static void addThreadStartTimeMetric(final String threadId,
         );
     }
 
+    public static void addThreadStateTelemetryMetric(final String threadId,
+                                                     final StreamsMetricsImpl streamsMetrics,
+                                                     final Gauge<Integer> threadStateProvider) {
+        streamsMetrics.addThreadLevelMutableMetric(
+                THREAD_STATE,
```

Comment:
nit: indention too deep (should only be 4 whitespace, not 8 -- should be an IDE setting)

Reply:
Ack, I think I need to copy settings from someone this is a common occurance with my PRs